### ISSUE TITLE
Instrument Settings are Inaccessible

### ISSUE THREAD

**Issue by MatejGolian** [musescore/MuseScore#28332]
### Issue type

Accessibility issue (e.g. for keyboard-only or screen reader users)

### Description with steps to reproduce

Hello,

I'm a NVDA / screen reader user and I'm having trouble changing instruments.
I can open the instrument settings from the Layout panel, but sadly the dialog/window that gets displayed once I do that is completely inaccessible. Inaccessible as in nothing gets spoken.
The attached image should hopefully be a screenshot of the dialog/window I'm talking about. It would be amazing if you could address this ASAP.

Many thanks.

### Supporting files, videos and screenshots

![Image](https://github.com/user-attachments/assets/57a81c8a-579d-4096-8193-4b74dd713bbd)

### What is the latest version of MuseScore Studio where this issue is present?

4.5.2.251141402

### Regression

I was unable to check

### Operating system

Windows 10

### Checklist

- [x] This report follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting bugs and issues
- [x] I have verified that this issue has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar issues
- [x] I have attached all requested files and information to this report
- [x] I have attempted to identify the root problem as concisely as possible, and have used minimal reproducible examples where possible

**Comment by MarcSabatella** [comment]
This may be an issue with certain screen readers only - other screen readers seem to work better, including JAWS and Orca (Linux).  See https://github.com/musescore/MuseScore/issues/25404

Meanwhile, you can access these same settings abnd controls via the Staff/Part properties panel (accessed using your system's equivalent of right-click).

**Comment by MatejGolian** [comment]
OK, so this is a duplicate then. Sorry for that. I haven't managed to find that issue before.
In my case JAWS does not perform much better at all.
Thanks for the alternative suggestion, but unfortunately I have no clue how to do it via the Staff/Part properties. If you have the time, perhaps a tutorial video would be nice. I can't find an element focusable via keyboard where pressing the right mouse button or its keyboard equivalent would give me the relevant options.
I also hoped that something like that would be available in the Part Manager, but it doesn't seem to be there either.
Feel free to close this issue any time. If need be we can continue discussing this over in that other thread.

**Comment by MarcSabatella** [comment]
Make a *range* selection on the staff you wish to change, and then Shift+F10 should display the context menu with "Staff/Part properties" as one of the items.  If you have further questions, please ask on the official Support forum at musescore.org.

**Comment by MatejGolian** [comment]
Thanks, that works. IMO it's unintuitive that one has to select a range first to get that option to appear - at least I would never have thought of doing that without your instructions, but it's easy to do, so all things considered it's fine.
Thanks again and just close this issue if you want. 🙂
